crates/tool_proto_build/src/main.rs

```diff
@@ -4,9 +4,10 @@ fn main() {
         .bytes(["."])
         .compile_protos(
             &[
-                "../api/proto/wire.proto",
```

Comment:
Is there a reason to re-order these? The wire protocol is independent of the module protocols so having it first makes sense to me

Reply:
It was just my instinct for alphabetical order at play here because I tend to find things easier if they are ordered alphabetically. But if it does not make sense from another perspective I can revert it...

Reply:
I did not consider that the order may matter for the purpose of compilation...

Reply:
I'm not sure that the order here matters but looking at the code behind that function call, it does maintain order using `Vec` and `for` through the process. I'd suggest making the semantic meaning clearer here rather than sorting alphabetically. How about something like

```
&[
    // Wire protocol for sending messages over the K2 transport
    "../api/proto/wire.proto",

    // Module protocols, as the content of wire messages
    "../api/proto/fetch.proto",
    // ...
]
```

Where the module messages can be sorted alphabetically. What do you think?

Reply:
Sounds good - changed it accordingly.